toolkit/imageconfigs/files/osguard/repart.d/14-root-a.conf

```diff
@@ -1,6 +1,6 @@
 [Partition]
 Type=root
 Label=root-a
-SizeMinBytes=10G
+SizeMinBytes=12G
```

Comment:
Please update in this script as well - https://github.com/microsoft/azurelinux/blob/3.0-dev/toolkit/scripts/generate-repartd.sh

`generate-repartd.sh` is what automatically creates these repart.d files

Reply:
Updated and reran the script to ensure the configs match the desired state.